tox.ini

```diff
@@ -12,7 +12,7 @@ deps =
     pytest-cov
 
 commands =
-    jwt: pip install -e .[jwt]
+    jwt: pip install -e .[cryptography,jwt]
```

Comment:
add `jwt: cryptography` to the `deps` section above per http://tox.readthedocs.io/en/latest/config.html#generating-environments-conditional-settings

Reply:
niiice ok

Reply:
fix?